tests/common/configuration/secret_model.rs

```diff
@@ -23,6 +23,10 @@ impl SecretModel {
         node_config_file_path
     }
 
+    pub fn empty() -> SecretModel {
```

Comment:
OK, though it's more conventional to mark constructor-like methods with `Self` in the return type.